packages/contentful-app-extensions/user-team-memberships/src/locations/Field.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import {
+  MultipleEntryReferenceEditor,
+  CustomEntityCardProps,
+  useEntity,
+} from '@contentful/field-editor-reference';
+import { entityHelpers } from '@contentful/field-editor-shared';
+import {
+  EntryCard,
+  MenuItem,
+  Badge,
+  Heading,
+  Paragraph,
+  Text,
+  EntityStatus,
+  EntryCardProps,
+} from '@contentful/f36-components';
+import { FieldExtensionSDK, Entry } from '@contentful/app-sdk';
+import { useSDK, useAutoResizer } from '@contentful/react-apps-toolkit';
+
+type CustomEntryCardProps = CustomEntityCardProps & { entity: Entry };
+type CardProps = {
+  actions: EntryCardProps['actions'];
+  onClick: CustomEntityCardProps['onEdit'];
+  status: EntityStatus;
+};
+type TeamCardProps = {
+  id: string;
+  role: string;
+  isInactive: boolean;
+} & CardProps;
+
+const TeamCard = ({
+  id,
+  actions,
+  isInactive,
+  role,
+  onClick,
+  status,
+}: TeamCardProps) => {
+  const { data: team } = useEntity<Entry>('Entry', id);
+  if (!team) {
+    return <EntryCard isLoading />;
+  }
+  return (
+    <EntryCard
+      contentType="Team"
+      actions={actions}
+      onClick={onClick}
+      status={status}
+    >
+      <Heading marginBottom="none">
+        {team.fields.displayName?.['en-US']}{' '}
+        {isInactive && <Badge variant="secondary">Inactive</Badge>}
```

Comment:
Hey, I wonder if it's better to show the inactive date instead of a badge the same way in squidex or leave the badge below the role with a label like "user inactive in this team" because a badge "inactive" beside the team name makes me think that the team is inactive following the hub UI

<img width="845" alt="Screenshot 2023-05-08 at 14 53 48" src="https://user-images.githubusercontent.com/16595804/236896333-ec4ba601-628d-47f6-a2b4-57f67db36f29.png">

Reply:
I've moved it to be below the role.